scene/gui/rich_text_label.cpp

```diff
@@ -5412,6 +5412,9 @@ void RichTextLabel::append_text(const String &p_bbcode) {
 			push_language(lang);
 			pos = brk_end + 1;
 			tag_stack.push_front("lang");
+		} else if (tag == "br") {
+			add_text("\n");
```

Comment:
```suggestion
			add_text("\r");
```
I think HTML `[br]` is supposed to force line break within paragraph, not start a new paragraph (and `\n` or `\r\n` are currently paragraph separators). This can have different effects if used in a bidirectional text.